connectors/src/pages/index.tsx

```diff
@@ -0,0 +1,49 @@
+import styles from "./index.module.css";
+import { type NextPage } from "next";
```

Comment:
We might need it in the future, so should we keep it as an example of how to implement it or just remove it and figure out later?

Reply:
Let’s remove. This is a classic next app